scripts/export-vmdk.sh

```diff
@@ -82,11 +85,6 @@ EOF
 # Disable sudo tty requirement
 sed -i.bak -e 's/^Defaults    requiretty/#Defaults    requiretty/g' $MNT/etc/sudoers
 
-# Ensure vagrant is able to detect guest as redhat
-# https://github.com/mitchellh/vagrant/blob/7caaffbf389fb298840eafb1edcdf554b977182e/plugins/guests/redhat/guest.rb#L7
-touch $MNT/etc/redhat-release
```

Comment:
This was added for Vagrantfiles that attempted to configure private networking.
Will need to verify if still required.

Example:
```
n.vm.guest = :redhat
n.vm.network "private_network", ip: 172.30.0.10
```